src/runner.js

```diff
@@ -1,4 +1,24 @@
- /* istanbul ignore next */
-require('babel-core/register');
+/* istanbul ignore next */
+var fs = require('fs');
+/* istanbul ignore next */
+var path = require('path');
+
+/* istanbul ignore next */
+var babelrc = fs.readFileSync(path.resolve(__dirname, '../.babelrc'));
```

Comment:
How about try to find 'why babel can not locate babelrc file'?

Maybe try to use ```require('babel-register') ``` instead of ```require('babel-core/register')```

Because it's a bit look like hack.

But if you not find better solution, it's ok :)

Or what you fix with this ```babelrc``` file?